server/sonar-web/src/main/js/apps/marketplace/style.css

```diff
@@ -0,0 +1,33 @@
+.marketplace-editions {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
```

Comment:
remove wrapping:

![image](https://user-images.githubusercontent.com/2317341/31614872-06c28646-b288-11e7-9f33-179d9271cdfc.png)

Reply:
I added wrapping in case more editions are added in the future. But it seems that it's not working as expected...